packages/thirdweb/src/wallets/manager/index.ts

```diff
@@ -140,13 +140,9 @@
       }
     })();
 
-    // add personal wallet to connected wallets list
+    // add personal wallet to connected wallets list even if it's not the active one
     addConnectedWallet(wallet);
 
-    if (wallet.id !== "smart") {
```

Comment:
hmmm i think this check is important. you sure we can remove this? basically we never want to write 'smart' as the LAST_ACTIVE_EOA_ID